privacyidea/lib/user.py

```diff
@@ -301,13 +301,11 @@ def info(self):
         :return: a dict with all the userinformation
         :rtype: dict
         """
-        if self.is_empty():
+        if self.is_empty() or uid = None:
```

Comment:
This is too early. Uid does not exist, yet.
It is only returned in line 307. Thus I recommend returning an empty dict in line 310.

Reply:
I recognized that myself and fixed it with d5b4acc, about five minutes after submitting ff8300b.